src/installer.ts

```diff
@@ -155,7 +155,9 @@ async function downloadRelease(version: string): Promise<string> {
   try {
     downloadPath = await tc.downloadTool(downloadUrl);
   } catch (error) {
-    core.debug(error);
+    if (typeof error === "string") {
+      core.debug(error);
+    }
```

Comment:
@per1234 I think the type of `error` is usually "Error".

If you want to support both Errors and strings, this would do the trick

```suggestion
    if (typeof error === "string" || error instanceof Error) {
      core.debug(error.toString());
    }
```

Reply:
Wouldn't this work too? 🤔 

```
  } catch (error: string | Error) {
```

> To get around this, you can specifically add runtime checks to ensure that the thrown type matches your expected type. Otherwise, you can just use a type assertion, add an explicit : any to your catch variable, or turn off --useUnknownInCatchVariables.

From: https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#using-unknown-in-catch-variables

Reply:
> I need a little bit of context here:

Unfortunately, I don't have much. This code was written by someone [years ago](https://github.com/arduino/setup-task/blob/3fcc6951abcb03b78448fb1b17ef0273093265bc/src/installer.ts#L70) (notice they were throwing a string back then). I've tried to leave it as is as much as possible, while focusing on improving the maintainability of this project we rely on so much, but also had neglected. I saw that the parameter type of `core.debug()` [is `string`](https://github.com/actions/toolkit/blob/ea81280a4d48fb0308d40f8f12ae00d117f8acb9/packages/core/src/core.ts#L227), so I stuck with that, but I also thought it was odd. I don't see any possible harm in printing debug output for `Error` types as well so I think it's a good change, which I have now made:
https://github.com/arduino/setup-task/compare/748085daba3320d8885e102e023569fd2b96d1e1..e7ea1aa3ef418a45b4a7f625ab861fc7f4f3c581
Thanks so much for your input!

Reply:
>Wouldn't this work too? 🤔

I don't believe so:
```
error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.
```
This is the thing that I struggled with while trying to learn how to fix the errors.

Reply:
We can merge this I guess, should I approve or it risks being automatically merged?

Reply:
OK, I went ahead and merged it. Thanks for your input!

One thing I'm noticing is that updating the same dependency in multiple projects at the same time is quite a bit of an improvement in efficiency because I can apply whatever I learned from the first update to all the others. Since all our TypeScript actions other than `arduino/create-changelog` have essentially the same code, it would be great to get them all synced up.